driver/vhci/vhci_dev.h

```diff
@@ -128,6 +128,7 @@ typedef struct
 	LONG	InterfaceRefCount;
 	// a pending irp when no urb is requested
 	PIRP	pending_read_irp;
+	BOOLEAN	pending_read_irp_cancellable;
```

Comment:
Is this variable required? It is used only in `complete_pending_read_irp()`. It would be no problem even though  `complete_pending_read_irp()` always clears a cancel routine irregardless of that variable. Because the pending read irp will be completed right after clearing. And there seems to be no possibility that a cancel routine other than `on_pending_irp_read_cancelled` intervenes. A pending read irp stays in a vhci driver.

Reply:
removed, unneeded